packages/common/android/src/oldarch/com/reactnativevectoricons/common/NativeVectorIconsSpec.java

```diff
@@ -0,0 +1,43 @@
+
+/**
+ * This code was generated by [react-native-codegen](https://www.npmjs.com/package/react-native-codegen).
+ *
+ * Do not edit this file as changes may cause incorrect behavior.
+ *
+ * Instead turn on new architecture, make JS spec file changes, re-run codegen (you can trigger that by rebuilding) and copy codegen result here.
+ *
+ * @generated by codegen project: GenerateModuleJavaSpec.js
+ *
+ * @nolint
+ */
+
+package com.reactnativevectoricons.common;
+
+import com.facebook.proguard.annotations.DoNotStrip;
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.turbomodule.core.interfaces.TurboModule;
+import javax.annotation.Nonnull;
+
+public abstract class NativeVectorIconsSpec extends ReactContextBaseJavaModule implements TurboModule {
```

Comment:
in both new and old architecture builds, `VectorIconsModule` subclass this file. In new arch, it's generated at build time. For old arch, it lives here. It looks identical to what codegen generates when new arch is on